### PR TITLE
Use Github Actions instead of Travis

### PR DESCRIPTION
**Pull request by pepakriz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +0,0 @@
-language: php
```
Comment (VincentLanglet) [inline .travis.yml:1]: In `.coverall.yml`

```
service_name: travis-ci
coverage_clover: tests/tmp/clover.xml
json_path: tests/tmp/coveralls.json
```

Should we rename the service ? @pepakriz

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+on: [pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
```
Comment (VincentLanglet) [inline .github/workflows/tests.yml:11]: You'll need to handle the conflict with https://github.com/pepakriz/phpstan-exception-rules/pull/126 and drop the 7.1 support.

File: .travis.yml (hunk 1 above)
Comment (pepakriz) [reply .travis.yml:1]: Coverall integration is not great currently. I'm thinking to remove it in this PR and make a better solution as a separate PR.

Comment (VincentLanglet) [reply .travis.yml:1]: Ok !

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**